File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    answer = 0
    alpha = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for i in range(len(alpha)):
        if s.find(alpha[i]) >= 0:
            s = s.replace(alpha[i], str(i))
    answer = int(s)
    return answer

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
    public int solution(String s) {
        String[] alpa = new String[]{"zero", "one", "two", "three", "four",
        "five", "six", "seven", "eight", "nine"};

        for (int i = 0; i < alpa.length; i++) {
            s = s.replace(alpa[i], String.valueOf(i));
        }

        return Integer.parseInt(s);
    }
}